### DISCUSSION TITLE
Serial port configuration ?

### DISCUSSION THREAD

**Discussion by anneranch** [blueman-project/blueman#2284] [Q&A] [answered]
Is there a way to use Blueman to configure serial port?
I am using Blueman to connect to a device which requires  
specific serial port configuration
- baud rate etc.

**Comment by cschramm** [comment, marked as answer]
The SerialManager plugin allows to provide a script path in its settings that it will call upon connection. Most probably you'll want to do an `stty` call there, passing the device file from the fifth argument and possibly conditional on the device address in the first argument.
